tests/database/test_uniprot.py

```diff
@@ -0,0 +1,33 @@
+# This source code is part of the Biotite package and is distributed
+# under the 3-Clause BSD License. Please see 'LICENSE.rst' for further
+# information.
+
+import itertools
+import tempfile
+import pytest
+import biotite.database.uniprot as uniprot
+import biotite.sequence.io.fasta as fasta
+from ..util import cannot_connect_to
+
+
+UNIPROT_URL = "https://www.uniprot.org/"
+
+
+pytest.mark.skipif(
```

Comment:
`skipif()` is a decorator, so you need to add an ``@`` before it to take effect. Furthermore, I think the two empty lines after the `skipif()` statement should be removed, so it gets clearer, that the decorator is associated to `test_fetch()`.